backend-api/openApiSpecs/async-public-spec.yaml

```diff
@@ -34,7 +34,7 @@ paths:
   /async/activeSession:
     get:
       security:
-        - bearerAuth: [ ]
+        - bearerAuth: []
       summary: Look for an active session for a given user
       description: Query the session database to find an active session for a given user. If an active session is found it returns the sessionId if not return 404 session not found
```

Comment:
I was wondering if it's worth adding below `description`, on line 40,  the `authorization` header parameter to make it clear that one is required? For example:

    parameters:
      - name: Authorization
        in: header
        required: true
        description: "A valid service token (e.g.: Authorization: Bearer < service-token-value>)"
        schema:
          type: string

Reply:
@anahickmann added, could you double check it and also the indentation?

Reply:
Looks good to me - thank you!